src/Turbo/src/Twig/TurboStreamListenRendererInterface.php

```diff
@@ -21,7 +21,8 @@
 interface TurboStreamListenRendererInterface
 {
     /**
-     * @param string|object $topic
+     * @param string|object       $topic
+     * @param array<string,mixed> $eventSourceOptions
      */
-    public function renderTurboStreamListen(Environment $env, $topic): string;
+    public function renderTurboStreamListen(Environment $env, $topic, array $eventSourceOptions = []): string;
```

Comment:
You cannot add parameter on an interface, even with default values, without breaking existing implementation.


```php
interface FooInterface
{
    public function bar(string $a, array $eventSourceOptions = []): string;
}

class AppFoo implements FooInterface
{
    // existing method matching previous interface
    public function bar(string $a): string
    {
        return $a;
    }
}
```

This throws a PHP Fatal error as a the concrete implementation is not compatible anymore ([see here](https://3v4l.org/0elpc)).

```
Fatal error: Declaration of AppFoo::bar(string $a): string must be compatible with FooInterface::bar(string $a, array $eventSourceOptions = []): string in php-wasm run script on line 11
```

So to make this change in a BC way, you need:
* to comment the new argument in the interface
* to use func_get_args to fetch the value in the concrete class

You can see here a recent example of Symfony PR with similar BC layer: https://github.com/symfony/symfony/pull/54531

Poke me if you need a hand here!